master/internal/experiment_job_service.go

```diff
@@ -36,8 +36,10 @@ func (e *internalExperiment) ToV1Job() (*jobv1.Job, error) {
 		WorkspaceId:    int32(workspace.ID),
 	}
 
-	j.IsPreemptible = config.ReadRMPreemptionStatus(j.ResourcePool)
-	j.Priority = int32(config.ReadPriority(j.ResourcePool, &e.activeConfig))
```

Comment:
I think there is a preexisting bug here

j.ResourcePool is set on line 45? So this will always be the empty string?